src/language/jhi-translate.directive.ts

```diff
@@ -16,8 +16,10 @@
  See the License for the specific language governing permissions and
  limitations under the License.
  */
-import { Input, Directive, ElementRef, OnChanges, OnInit, Optional } from '@angular/core';
+import { Input, Directive, ElementRef, OnChanges, OnInit, Optional, OnDestroy } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
+import { ReplaySubject } from 'rxjs';
```

Comment:
you don't need `ReplaySubject` here. Code should be changed to use `Subject`

Reply:
It's already merged, but I can create another PR for this.

Reply:
See #114.